zebra-chain/src/orchard/tree.rs

```diff
@@ -153,9 +155,43 @@ impl ZcashDeserialize for Root {
 }
 
 /// A node of the Orchard Incremental Note Commitment Tree.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Copy)]
 struct Node(pallas::Base);
 
+impl merkle_tree::Hashable for Node {
```

Comment:
Optional: add comment explaining why this is needed (e.g. "Required to convert a NoteCommitmentTree into a SerializedTree")